Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -301,5 +309,228 @@ configuring the CiliumLocalRedirectPolicy.
 Local Redirect Policy updates are currently not supported. If there are any
 changes to be made, delete the existing policy, and re-create a new one.
 
+Use Cases
+=========
+Local redirect policy allows Cilium to support the following use cases:
+
+Node-local DNS cache
+--------------------
+`DNS node-cache <https://github.com/kubernetes/dns>`_ listens on a static IP to intercept
+traffic from application pods to the cluster's DNS service VIP by default, which will be
+bypassed when Cilium is handling service resolution at or before the veth interface of the
+application pod. To enable the DNS node-cache in a Cilium cluster, the following example
+steers traffic to a local DNS node-cache which is run as a normal pod.
+
+* Deploy DNS node-cache in pod namespace.
+
+  .. tabs::
+
+    .. group-tab:: Quick Deployment
+
+        Deploy DNS node-cache.
+
+        .. parsed-literal::
+
+            $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/node-local-dns.yaml
```

Comment:
These changes don't work on GKE since it doesn't let users deploy pods in the `kube-system` namespace. @Weil0ng Please document the expected workflow.

Reply:
Also, I tried this on a non-GKE cluster. I'm unable to download the image. 

 ```
 Normal   Scheduled       34s                default-scheduler  Successfully assigned kube-system/node-local-dns-gdn86 to k8s1
  Normal   Pulling         21s (x2 over 33s)  kubelet, k8s1      Pulling image "k8s.gcr.io/dns/k8s-dns-node-cache:1.15.16"
  Warning  Failed          20s (x2 over 32s)  kubelet, k8s1      Failed to pull image "k8s.gcr.io/dns/k8s-dns-node-cache:1.15.16": rpc error: code = Unknown desc = Error response from daemon: manifest for k8s.gcr.io/dns/k8s-dns-node-cache:1.15.16 not found: manifest unknown: Failed to fetch "1.15.16" from request "/v2/dns/k8s-dns-node-cache/manifests/1.15.16".
  Warning  Failed          20s (x2 over 32s)  kubelet, k8s1      Error: ErrImagePull
  Normal   SandboxChanged  18s (x7 over 31s)  kubelet, k8s1      Pod sandbox changed, it will be killed and re-created.
  Normal   BackOff         16s (x6 over 29s)  kubelet, k8s1      Back-off pulling image "k8s.gcr.io/dns/k8s-dns-node-cache:1.15.16"
  Warning  Failed          16s (x6 over 29s)  kubelet, k8s1      Error: ImagePullBackOff
```

Reply:
GKE won't let you deploy in kube-system ns, I added a note about this

The image error is because the new release version 1.15.16 has not been pushed to the bucket yet by the kubernetes dns maintainers, question here is what do we do if the release is still not available when we release 1.9? Do we add a note or? Thoughts?

Reply:
/cc @joestringer @aanm

Reply:
Assuming the latest update from @Weil0ng on Slack about an upstream release coming later this week, I'm okay to proceed with this as-is. We can just mention that this part doesn't yet work to the person validating this GSG, then do a quick check once the upstream release is out to confirm that the instructions work and don't need any further changes.

Reply:
I confirm that the image is now publicly available.